### ISSUE TITLE
Does VSC really need to be added as a game?

### ISSUE THREAD

**Issue by Xalaxis** [maxerbox/vscode-discord#6]
On my copy of Discord VSC is not added and yet the rich presence shows.

**Comment by maxerbox** [comment]
You're right, Visual Studio Code doesn't need to be added in "Games" on discord. Such a thing is just to show a simple "Playing Visual Studio Code " when the process is running.

**Comment by fratik** [comment]
In the newest Discord, you don't need to make VSCode as a game. It works automatically (at least on my PC)

**Comment by maxerbox** [comment]
See https://github.com/maxerbox/vscode-discord/issues/7

**Comment by Xalaxis** [comment]
![image](https://user-images.githubusercontent.com/12937964/33240422-a44c076c-d2ad-11e7-8608-c3aef18630f5.png)
VSCode seems to have been automatically added as a game by Discord now.  Maybe they added it to their server-side database? Of note is that the 'game' has been given no name.  I **have not** added this manually.

**Comment by maxerbox** [comment]
That's due to the Windows Registering, see https://github.com/maxerbox/vscode-discord/issues/15

**Comment by Xalaxis** [comment]
Why did it get removed if it was a useful feature?  As I understand it, removing the game from Discord's list breaks the removal of 'Playing vscode' from Discord.

**Comment by maxerbox** [comment]
Well, I followed the reviews on the markplace:
https://marketplace.visualstudio.com/items?itemName=maxerbox.vscode-discord#review-details

2017-11-26 17:14 GMT+01:00 Matthew Wolffsohn <notifications@github.com>:
